insights/client/__init__.py

```diff
@@ -19,7 +19,7 @@
 
 LOG_FORMAT = ("%(asctime)s %(levelname)s %(message)s")
 APP_NAME = constants.app_name
-logger = logging.getLogger(__name__)
+logger = logging.getLogger(APP_NAME)
```

Comment:
I don't see how having each module get the same logger is any better than getting a module-specific one.  You may as well just do `logger = logging.root`.